pkg/codegen/schema/schema.go

```diff
@@ -431,6 +433,8 @@ type PackageSpec struct {
 	Homepage string `json:"homepage,omitempty"`
 	// License indicates which license is used for the package's contents.
 	License string `json:"license,omitempty"`
+	// Attribution allows freeform text attribution of derived work, if needed.
+	Attribution string `json:"attribution,omitempty"`
```

Comment:
Given how we're using this - I might just rename this to `Notes` to be more generic.

Reply:
To me `Notes` sounds specific to how we show this information. If the information in this property is really attribution, then I'd suggest leaving it as-is, otherwise, yeah `Notes` would be good here.